fastlane_core/spec/configuration_file_spec.rb

```diff
@@ -57,6 +57,30 @@
         expect(config[:app_identifier]).to eq("detlef.app.super")
       end
 
+      it "supports modifying of frozen strings too" do
+        # Test that a value can be modified (this isn't the case by default if it's set via ENV)
+        app_identifier = "com.krausefx.yolo"
+        ENV["SOMETHING_RANDOM_APP_IDENTIFIER"] = app_identifier
+        config = FastlaneCore::Configuration.create(options, {})
+        config.load_configuration_file('ConfigFileEnv')
+        expect(config[:app_identifier]).to eq(app_identifier)
+        config[:app_identifier].gsub!("yolo", "yoLiveTwice")
+        expect(config[:app_identifier]).to eq("com.krausefx.yoLiveTwice")
+        ENV.delete("SOMETHING_RANDOM_APP_IDENTIFIER")
```

Comment:
For unit tests we have a helper, `with_env_values` that will ensure that we don't forget to delete the ENV value at the end. Let's keep using that 👍 

```ruby
with_env_values('SOMETHING_RANDOM_IOS_VERSION' -> ios_version) do
  config = FastlaneCore::Configuration.create(options, {})
  config.load_configuration_file('ConfigFileEnv')
  expect(config[:ios_version]).to eq(ios_version)
  config[:ios_version].gsub!(".1", ".2")
  expect(config[:ios_version]).to eq("9.2")
end
```